docs/pages/components/checkbox.md

```diff
@@ -0,0 +1,310 @@
+                Text Option
+            </label>
+        </div>
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox is-information" id="Ai4ez61294" checked>
+            <label class="fd-checkbox__label" for="Ai4ez61294">
+                Selected State
+            </label>
+        </div>
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox is-information" id="Ai4ez613i4">
+            <label class="fd-checkbox__label" for="Ai4ez613i4">
+                TriState Text
+            </label>
+        </div>
+    </div>
+</fieldset>
+
+<fieldset class="fd-fieldset">
+    <legend class="fd-fieldset__legend">Checkboxes Disabled</legend>
+    <div class="fd-form-group">
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox" id="Ai4ez61196" disabled>
+            <label class="fd-checkbox__label" for="Ai4ez61196">
+                Text Option
+            </label>
+        </div>
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox" id="Ai4ez61296" checked disabled>
+            <label class="fd-checkbox__label" for="Ai4ez61296">
+                Selected State
+            </label>
+        </div>
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox" id="Ai4ez613i6" disabled>
+            <label class="fd-checkbox__label" for="Ai4ez613i6">
+                TriState Text
+            </label>
+        </div>
+    </div>
+</fieldset>
+
+<fieldset class="fd-fieldset" dir="rtl">
+    <legend class="fd-fieldset__legend">Checkboxes RTL Support</legend>
+    <div class="fd-form-group">
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox" id="Ai4ez611tt">
+            <label class="fd-checkbox__label" for="Ai4ez611tt">
+                Text Option
+            </label>
+        </div>
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox" id="Ai4ez612tt" checked>
+            <label class="fd-checkbox__label" for="Ai4ez612tt">
+                Selected State
+            </label>
+        </div>
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox" id="Ai4ez613i7">
+            <label class="fd-checkbox__label" for="Ai4ez613i7">
+                TriState Text
+            </label>
+        </div>
+    </div>
+</fieldset>
+{% endcapture %}
+
+{% include display-component.html component=checkbox %}
+
+## Display Only Checkbox
+{% capture display-checkbox %}
+<fieldset class="fd-fieldset">
+    <legend class="fd-fieldset__legend">Display Only Checkboxes</legend>
+    <div class="fd-form-group">
+        <div class="fd-form-item">
+            <input type="checkbox" class="fd-checkbox fd-checkbox--display-only" id="1tn4O446" checked>
+            <label class="fd-checkbox__label--display-only" for="1tn4O446">Label</label>
```

Comment:
These should also have class `fd-checkbox__label`:  `fd-checkbox__label fd-checkbox__label--display-only`